src/main/java/com/work/management/logging/LogRequestBodyAdviceAdapter.java

```diff
@@ -0,0 +1,35 @@
+package com.work.management.logging;
+
+import com.work.management.service.log.LoggingService;
+import java.lang.reflect.Type;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.MethodParameter;
+import org.springframework.http.HttpInputMessage;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.servlet.mvc.method.annotation.RequestBodyAdviceAdapter;
+
+@ControllerAdvice
+final class LogRequestBodyAdviceAdapter extends RequestBodyAdviceAdapter {
+
+  @Autowired
+  private LoggingService loggingService;
+
+  @Autowired
+  private HttpServletRequest httpServletRequest;
+
+  @Override
```

Comment:
make loggingService and httpServletRequest private final and use Constructor injection instead of field injection.